### PR TITLE
Add new spelling correction for "certifciate" and add additional

### PR DESCRIPTION
**Pull request by cfi-gb**
variants for existing ones.

See e.g. https://github.com/search?q=certifciate&type=Code

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -3936,6 +3936,7 @@ cavaets->caveats
 ccahe->cache
 ccale->scale
 ccertificate->certificate
+ccertificates->certificates
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Add ccertificated->certificated and ccertification->certification

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -4016,10 +4017,16 @@ certifactes->certificates
...
+certifciate->certificate
+certifciates->certificates
 certificat->certificate
+certificats->certificates
 certifice->certificate
+certifices->certificates
+certifificate->certificate
 certifificates->certificates
 certiticate->certificate
+certiticates->certificates
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Can you add the -ed and -ion variants of these ones too please?

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -6713,6 +6720,7 @@ cryptograpic->cryptographic
 crystalisation->crystallisation
 cryto->crypto
 crytpo->crypto
+ctificate->certificate
 ctificates->certificates
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Again

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (cfi-gb) [reply codespell_lib/data/dictionary.txt]: 6b5626ca558be44bc06f48e61e85c44dff74bd06

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (cfi-gb) [reply codespell_lib/data/dictionary.txt]: 6b5626ca558be44bc06f48e61e85c44dff74bd06

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (cfi-gb) [reply codespell_lib/data/dictionary.txt]: 6b5626ca558be44bc06f48e61e85c44dff74bd06

**Comment by cfi-gb** [comment]
Sure, thanks for the feedback / review. Added these variants in 6b5626ca558be44bc06f48e61e85c44dff74bd06, hope i got all variants.

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -4006,20 +4009,42 @@ certaint->certain
...
+certifcated->certificated
 certifcates->certificates
 certifcation->certification
+certifciate->certificate
+certifciated->certificated
+certifciates->certificates
+certifciation->certification
 certificat->certificate
+certificatd->certificated
+certificats->certificates
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: I think you've missed `certificats->certification`

Comment (cfi-gb) [reply codespell_lib/data/dictionary.txt]: `certificats->certification` doesn't look right for me, do you had something else in mind and was this just a copy'n'paste mistake?

Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: Indeed I did, I failed to edit it properly, although now looking at it closely, I see they've all dropped the e, which obviously doesn't exist in certification.

Up to you if you think it's worth adding these two:
```
certificatin->certification
certificaton->certification
```

Although if you do, the former should probably suggest certificating first.

Comment (cfi-gb) [reply codespell_lib/data/dictionary.txt]: Thanks again. Both seems to be quite common:

https://github.com/search?q=certificatin&type=Code (178 code results)
https://github.com/search?q=certificaton&type=Code (651 code results) 

Updated / added in 305693b9f278a3c9340389e4c944f3e044c02b74

**Pull request merged**